pytest_mpl/plugin.py

```diff
@@ -63,6 +64,14 @@
 ALL_IMAGE_FORMATS = RASTER_IMAGE_FORMATS + VECTOR_IMAGE_FORMATS
 
 
+def _get_item_dir(item):
+    if Version(pytest.__version__) < Version("7.0.0"):
+        path = Path(item.fspath)
+    else:
+        path = item.path
```

Comment:
Do you think this would be sufficient?
```suggestion
    path = getattr(item, "path", Path(item.fspath))
```
Just trying to avoid parsing and comparing the versions. If you think checking the version is better can you do the comparison in a constant at the top of the file.

Reply:
This helper function is only needed because older versions of `pytest` are too different from the recent ones, and having the version check made that explicit. But conveying that in a comment makes the code shorter and removes the need to import `Version`, so I have adopted your suggestion.

Reply:
This simple `getattr()` call does not work. Sadly I realized this only after this pull request was already merged, so I had to open a follow-up pull request: #213.